www/app/templates/about.hbs

```diff
@@ -4,6 +4,7 @@
   </div>
   <h3>Terms of Service</h3>
   <p>By using the pool you accept all possible risks related to experimental software usage.<br/>
-     Pool owner can't compensate any irreversible losses, but will do his best to prevent worst case.
+     Pool owner can't compensate any irreversible losses, but will do his best to prevent worst case.<br/>
+     This is official pool of <a href="https://mintme.com/coin" rel="nofollow" target="_blank">mintme.com/coin</a>, for any queries, please contact us on official email visible on our main page.
```

Comment:
as i see, we don't have email on the main page, but i can be wrong, pls confirm this

Reply:
I don't think there is, but I added that line because it's there on https://www.mintme.com/pool/#/about

Reply:
So this is not on the main page, you should change the text to `about page`